setup.py

```diff
@@ -14,8 +14,8 @@
 test_requirements = []
 
 setup(
-    author="Marco Gorelli",
-    author_email="m.e.gorelli@gmail.com",
+    author="Marco Gorelli, Girish Pasupathy",
+    author_email="",
```

Comment:
Since `""` is the default for `author_email`, I guess you can just drop the line 😉 .
Or if @girip11 is fine with it add both your emails, if that is wanted.